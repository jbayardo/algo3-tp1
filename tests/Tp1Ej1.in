6
6 8 12 15
35
8 14 20 40 45 54 60 67 74 89 99
100
35 87 141 163 183 252 288 314 356 387
90
6 8 16 19 28 32 37 45 52 60 69 78 82
4
5 13 19 26 35
5
5 13 19 26 35
5
7 16 19 27 33
8
2 5 8 14 18
8
3 6 9 15 19