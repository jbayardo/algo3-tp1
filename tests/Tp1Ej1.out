3
5
3
14
0
2
2
4
3